code/ai/aicode.cpp

```diff
@@ -14219,15 +14212,7 @@ void ai_process( object * obj, int ai_index, float frametime )
 	if (rfc) {
 		// Wanderer - sexp based override goes here - only if rfc is valid though
 		ai_control_info_check(aip, sip, &obj->phys_info);
-		vec3d copy_desired_rotvel = obj->phys_info.rotvel;
 		physics_read_flying_controls( &obj->orient, &obj->phys_info, &AI_ci, frametime);
-		// if obj is in formation and not flight leader, don't update rotvel
-		if (aip->ai_flags[AI::AI_Flags::Formation_object, AI::AI_Flags::Formation_wing]) {
-			if (&Objects[aip->goal_objnum] != obj) {
-				obj->phys_info.desired_rotvel = copy_desired_rotvel;
-				obj->phys_info.rotvel = copy_desired_rotvel;
-			}
-		}
```

Comment:
Why, at the very end it feels the need to copy its rotvel, even though `read_flying_controls` does nothing to rotvel, or why it _also_ copies that rotvel into desired_rotvel even though the rest of the AI code doesn't use desired_rotvel and now allows it to turn much faster than normal, I couldn't tell you. This check doesn't even catch exclude all wingleaders because often AI wingleaders won't even have their goal_objnum set. And probably most importantly completely overrides the maneuver overrides!